content/6_Advanced/LCT.mdx

```diff
@@ -355,16 +388,226 @@ int main() {
 }
 ```
 
-## Link Cut Tree - Connectivity
+## Finding Connectivity
+
+The link cut tree can be used to solve problems that deal with queries updating trees and querying for connectivity.
+
+### Problems
 
 <Problems problems="con" />
 
 ## Link Cut Tree - Paths
 
-<Problems problems="pathSam" />
+<FocusProblem problem="pathSam" />
+
+### Explanation
+
+Link Cut Tree can also calculate the aggregate (max, min, sum, etc.) of the weights of the edges or nodes on a path. To do this, we can define a function that will return an aggregate for the path from the root of the tree to the provided vertex. We can augment the auxiliary splay trees with the value(s) we want to keep track of. The aggregate for the path from the root of a tree to a vertex can be found by retrieving the value(s) from the splay tree created after accessing the vertex.
 
 ### Implementation
 
+**Time Complexity:** $\mathcal{O}(?)$
+
+```cpp
```

Comment:
around all cpp code, please wrap it with `<LanguageSection>` and `<CPPSection>`
check the other modules for how they do stuff

Reply:
Done